### DISCUSSION TITLE
raspicat_documentation/chat_corner/memo/

### DISCUSSION THREAD

**Discussion by giscus** [CIT-Autonomous-Robot-Lab/raspicat_documentation#15] [Announcements]
# raspicat_documentation/chat_corner/memo/

https://uhobeike.github.io/raspicat_documentation/chat_corner/memo/

**Comment by TonoLeo** [comment]
https://mebee.info/2021/05/26/post-34983/

**Comment by TonoLeo** [comment]
sudo apt install build-essential

**Reply by uhobeike** [reply]
記事に追加しておきます。

**Comment by RyodaiArai** [comment]
https://server-network-note.net/2021/05/ubuntu-2004-lts-server-jst-timezone/

**Reply by uhobeike** [reply]
記事に追加しておきます。

**Comment by uhobeike** [comment]
やること
・ウェイポイントナビゲーションの方法を書く
・RVizのアイコンの修正
・RVizの操作をC++でやる
・python両方使えるようにする

**Reply by uhobeike** [reply]
あと、記事の修正

**Comment by TonoLeo** [comment]
リモート接続
ip:10.42.0.178
raspi側をマスター
ROS_MASTRE_URI=http://192.168.12.1:11311
※自分のipも変わる

**Comment by TonoLeo** [comment]
http://wiki.ros.org/velodyne/Tutorials/Getting%20Started%20with%20the%20Velodyne%20VLP16#Connecting_your_computer_to_LIDAR_through_terminal

**Comment by my9918** [comment]
wsl2
https://qiita.com/koichi_baseball/items/fbab3acbcd41b18bae2e

**Comment by TonoLeo** [comment]
roslaunch raspicat_navigation raspicat_bringup_navigation.launch waypoint_navigation:=true speak:=false usens_conv:=false urg_usb:=true urg_ether:=false mcl_init_pose_x:=0 mcl_init_pose_y:=0 mcl_init_pose_a:=0 csv_file:="$(rospack find raspicat_navigation)/config/csv/waypoint_19.csv" mcl_map_file:="$(rospack find raspicat_navigation)/config/maps/tsudanuma/map_19.yaml" move_base_map_file:="$(rospack find raspicat_navigation)/config/maps/for_move_base/map_19.yaml"

**Comment by TonoLeo** [comment]
~/raspicat_ws/src/raspicat_navigation/config/csv/

**Comment by TonoLeo** [comment]
nd=nodelist[0].decode("utf-8")

**Comment by TonoLeo** [comment]
roslaunch raspicat_navigation waypoint_set.launch csv_file:="$(rospack find raspicat_navigation)/config/csv/" map_file:="l"

**Comment by TonoLeo** [comment]
roslaunch raspicat_navigation waypoint_set.launch csv_file:="$(rospack find raspicat_navigation)/config/csv/" map_file:=""

**Comment by TonoLeo** [comment]
# Raspberry Pi 4B
roslaunch raspicat_navigation raspicat_bringup_navigation.launch waypoint_navigation:=true speak:=false usens_conv:=false urg_usb:=true urg_ether:=false mcl_init_pose_x:=0 mcl_init_pose_y:=0 mcl_init_pose_a:=0 csv_file:="$(rospack find raspicat_navigation)/config/csv/waypoint.csv" mcl_map_file:="" move_base_map_file:=""

**Comment by AtsukiUsui** [comment]
Terminatorを使ってるB3各位へ

コマンドを全ターミナルにbroadcast
他にも、source ~/.bashrcや、環境変数の設定を分割した全ターミナルに反映させたいときは下記が役立ちます.
broadcast_all : 全ターミナルにコマンド入力(alt+A)
brodcast_group : 現在のターミナルのみにコマンド入力(alt+O)

参考URL：https://cryborg.hatenablog.com/entry/2016/09/03/164940
